src/ucp/core/ucp_mm.c

```diff
@@ -1399,15 +1399,14 @@ static ucs_rcache_ops_t ucp_mem_rcache_ops = {
 
 static ucs_status_t
 ucp_mem_rcache_create(ucp_context_h context, const char *name,
-                      ucs_rcache_t **rcache_p)
+                      ucs_rcache_t **rcache_p, const ucs_rcache_config_t *config)
 {
     ucs_rcache_params_t rcache_params;
 
+    ucs_rcache_set_params(config, &rcache_params);
```

Comment:
seems like many rcache parameters are overridden in the next lines: alignment, flags, event priority.
better override their default values in ucp_context.c:550, like done in gdr_copy_md.c:33 for example

Reply:
@yosefe I tried along these lines:
```C
 549      
 550   {"", "RCACHE_MEM_PRIO=500", NULL,
 551    ucs_offsetof(ucp_config_t, rcache_config),
 552    UCS_CONFIG_TYPE_TABLE(ucs_config_rcache_table)},
 553      
 554   {"", "RCACHE_PURGE_ON_FORK=y", NULL,
 555    ucs_offsetof(ucp_config_t, rcache_config),
 556    UCS_CONFIG_TYPE_TABLE(ucs_config_rcache_table)},
```
but this results in the last entry overwriting previous initialization. Is there a way to combine the all initialization?

Reply:
it should be ""RCACHE_PURGE_ON_FORK=y;RCACHE_MEM_PRIO=500"
see https://github.com/openucx/ucx/blob/b533b2802bd858f3d768fe01a7b68e8ccb23f70d/src/uct/ib/dc/dc_mlx5.c#L47